draft-ietf-rtcweb-jsep.xml

```diff
@@ -1766,8 +1766,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     defined for Trickle ICE.
                     If the MID, m-line index, or candidate string provided in the ICE candidate is
                     invalid, an error is generated.
+                    If present, ufrag is used to determine which
+                    generation of remote description the candidate corresponds to;
```

Comment:
"Generation of remote description" or "generation of ICE candidates"? Multiple remote descriptions could have the same ufrag/password.

Also, a small side question: is an ICE "generation" defined anywhere? I've been using the term informally for a while, but I can't find an official definition of it; only one mention in Trickle ICE.